react/features/base/config/actions.ts

```diff
@@ -139,6 +139,12 @@ export function setConfig(config: IConfig = {}) {
             config.bosh = bosh;
         }
 
+        //Gracetech
```

Comment:
Please fix build issues.
Since we'll never merge back to master, please remove such comment

Reply:
fixed